content/rdi/rdi-faq.md

```diff
@@ -0,0 +1,74 @@
+---
+Title: F.A.Q.
+linkTitle: FAQ
+description: Questions and answers about RDI
+weight: 70
+alwaysopen: false
+categories: ["redis-di"]
+aliases:
+---
+
+## General
+
+**WHich use cases are supported by RDI?**
+RDI supports the Ingest and Write behind use cases.
+Ingest is used to keep Redis in sync with the application system of records database so that read queries can be done in speed and scale from Redis.
+Write behind is used to speed up application read and writes using Redis as front database while asynchronously updating a downstream database.
+**Is RDI available as part of Redis Cloud?**
+RDI is currently available as a self managed product. This means you will need to install RDI engine and Debezium Server on hosts or K8s pods that you manage. However, RDI can write the data to a Redis CLoud target.
+**Is RDI available for open source Redis?**
+RDI only works with Redis Enterprise.
+**What is the minimal version of Redis Enterprise for running RDI?**
+The minimal version is 6.2.18
+**What are the pre-requisites for running RDI?**
+RDI requires Redis Gears 1.2.6 installed on the Redis ENterprise cluster. RDI can install Redis Gears for you.
+For more specific requirements per use case see the `Ingest` and `Write Behind` sections of this document.
```

Comment:
You should probably use links for the references to the Ingest and Write Behind (should be Write-behind; need to correct that at some point).

```suggestion
**What are the pre-requisites for running RDI?**
RDI requires RedisGears 1.2.6 installed on the Redis Enterprise cluster. RDI can install RedisGears for you.
For more specific requirements per use case, see the `Ingest` and `Write Behind` sections of this document.
```

Reply:
done